theano/sandbox/cuda/cuda_ndarray.cu

```diff
@@ -1030,7 +1030,6 @@ CudaNdarray_TakeFrom(CudaNdarray * self, PyObject *args){
         PyObject * indices_float32 = NULL;
         indices_float32 = PyArray_View((PyArrayObject*)indices_obj,
                                                   float32_descr, NULL);
-        Py_DECREF(float32_descr);
```

Comment:
I checked numpy code and PyArray_DescrFromType do an incref. The question is, do PyArray_view decref it? I'm checking.

Did you tested caglar code without the c code of this op?